### PR TITLE
Circleci project setup

### PR DESCRIPTION
**Pull request by skaehub**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+orbs:
+  node: circleci/node@5.0.2
```
Comment (KaigaMunene) [inline .circleci/config.yml:6]: input the latest node version

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+jobs:
+  build-and-test:
+    docker:
+      - image: "cimg/base:stable"
+    steps:
+      - checkout
+      - node/install:
+          node-version: "12.22"
```
Comment (KaigaMunene) [inline .circleci/config.yml]: same for here the node version

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      - node/install-packages
+      # Setup tests and activate this comman
+      # - run:
+      #     command: npm run test
+      - run:
+          name: "Say hello"
+          command: "echo Hello, World!"
```
Comment (KaigaMunene) [inline .circleci/config.yml]: add a meaningful run message

File: .circleci/config.yml (hunk 1 above)
Comment (dalienst) [reply .circleci/config.yml:6]: 5.0.2 is the latest according to the orbs registry

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    docker:
+      - image: "cimg/base:stable"
+    steps:
+      - checkout
+      - node/install:
+          node-version: "18.8.0"
+      - node/install-packages
+      # Setup tests and activate this comman
+      # - run:
+      #     command: npm run test
```
Comment (KaigaMunene) [inline .circleci/config.yml]: uncomment this

Comment (dalienst) [reply .circleci/config.yml]: Okay

**Review by 3N61N33R — APPROVED** [review]
LGTM!

**Pull request merged**